egs/mgb2/asr/conf/tuning/train_rnn.yaml

```diff
@@ -0,0 +1,32 @@
+# network architecture
+# encoder related
+etype: vggblstmp     # encoder architecture type
+elayers: 4
+eunits: 1024
+eprojs: 1024
+subsample: "1_2_2_1_1" # skip every n frame from input to nth layers
+# decoder related
+dlayers: 1
+dunits: 1024
+# attention related
+atype: location
+adim: 1024
+aconv-chans: 10
+aconv-filts: 100
+
+# hybrid CTC/attention
+mtlalpha: 0.5
+
+# minibatch related
+batch-size: 35
+maxlen-in: 600  # if input length  > maxlen_in, batchsize is automatically reduced
+maxlen-out: 150 # if output length > maxlen_out, batchsize is automatically reduced
+
+# optimization related
+sortagrad: 0 # Feed samples from shortest to longest ; -1: enabled for all epochs, 0: disabled, other: enabled for 'other' epochs
+opt: adadelta
+epochs: 10
+patience: 3
+
+# scheduled sampling option
+sampling-probability: 0.0
```

Comment:
Hi, do you have result for this RNN configuration?

Reply:
Hello @Jzmo,
No, we only tried the transformer-based model

Reply:
Yes I think it better to add that since we have the configuration file. We can update later once we have better result.

Reply:
Right now we have an experiment running with ASR transformer and RNN as a language model. Once it finishes I will update the RESULTS.md

Reply:
Added results  with RNN lm